src/main/java/seedu/address/model/person/Phone.java

```diff
@@ -11,8 +11,8 @@ public class Phone {
 
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Phone numbers should only contain numbers, and it should be at least 3 digits long";
-    public static final String VALIDATION_REGEX = "\\d{3,}";
+            "Phone numbers should only contain numbers, and it should be between 3 digits and 8 digits long";
```

Comment:
Not sure if we should allow for phone numbers with 4-7 digits

Also what do y'all think are the consequences of rejecting all phone numbers less than 8 digits long?

Reply:
I think thats fair, I think 3 digit numbers are reserved for emergency services anyways